src/IntegrationTests/IntegrationTests/OpenTelemetryExporterSmokeTest.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Newtonsoft.Json;
+using IntegrationTests.Fixtures;
+using System.Net.Http;
+using System.Threading;
+using Xunit;
+using Xunit.Abstractions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace IntegrationTests
+{
+    public class OpenTelemetryExporterSmokeTest : IClassFixture<OpenTelemetryUsageApplicationFixture>
```

Comment:
Re:naming - This test is for OT _Trace_ Exporter. We may also eventually add a test for OT _Metrics_ Exporter and/or NR Metrics Telemetry SDK.

Reply:
will do